examples/colors256.nim

```diff
@@ -0,0 +1,19 @@
+import nimbox
+import os
+
+when isMainModule:
+  var nb = newNimbox()
+  defer: nb.shutdown()
+
+  nb.outputMode = out256
+
+#Colors are defined using xTerm numbers.
```

Comment:
X term numbers? That's a bit weird. if there is 256 of these colours then I think an enum would be better

Reply:
I agree. I'm packing out my house this weekend, but should have time to go through and enumerate them all by Wednesday.

Reply:
OK, I've done some reading to find a list of color names and it looks like an enum is not the best option because the exact colors vary slightly from terminal emulator to terminal emulator (gotta love 1980s tech). I have instead used a Subrange, which restricts the user from inputting something outside the printable range.